telluric/plotting.py

```diff
@@ -109,14 +109,14 @@ def plot(feature, mp=None, style_function=None, **map_kwargs):
 
     Parameters
     ----------
-    feature : Any object with __geo_interface__
+    feature : GeoVector
```

Comment:
This is not really the case, since it works with `telluric.vectors.GeoVector, telluric.features.GeoFeature, telluric.collections.BaseCollection` as it's said in other parts of the documentation. Or at least, that's the intention. Could you please check that I'm not mistaken?

Reply:
You are right. All of them can be plotted. But I was confused by using `__geo_interface__`. I just thought that arbitrary object which implements [\_\_geo_interface\_\_](https://gist.github.com/sgillies/2217756) can be plotted with these functions.

Reply:
Arbitrary objects with `__geo_interface__` can't be plotted, since we're making use of `.is_empty`, `.envelope` and other properties, so the docstring here was wrong. This should be `telluric.vectors.GeoVector, telluric.features.GeoFeature, telluric.collections.BaseCollection`.